mailbox/event/event-rabbitmq/src/main/java/org/apache/james/mailbox/events/EventDispatcher.java

```diff
@@ -80,4 +105,11 @@ void start() {
         return eventSerializer.toJson(event).getBytes(StandardCharsets.UTF_8);
     }
 
+    private OutboundMessage generateMessage(RoutingKeyConverter.RoutingKey routingKey, byte[] payload) {
+        AMQP.BasicProperties properties = new AMQP.BasicProperties.Builder()
```

Comment:
These AMQP properties are the same for each messages hence maybe it makes little sense to always recompute it. 

Couldn't this be a class field?